scheduler/bin/run-docker.sh

```diff
@@ -78,7 +78,7 @@ docker create \
     -e "COOK_DATOMIC_URI=${COOK_DATOMIC_URI}" \
     -e "COOK_LOG_FILE=log/cook-${COOK_PORT}.log" \
     -v ${DIR}/../log:/opt/cook/log \
-    cook-scheduler:latest
+    cook-scheduler:latest ${COOK_CONFIG}
```

Comment:
Can we set a default value for this (could just be empty string) instead of possibly having it undefined?